src/GitLabApiClient/Models/Variables/Request/UpdateProjectVariableRequest.cs

```diff
@@ -0,0 +1,50 @@
+using Newtonsoft.Json;
+
+namespace GitLabApiClient.Models.Variables.Request
+{
+    public class UpdateProjectVariableRequest
+    {
+        /// <summary>
+        /// The ID of the project
+        /// </summary>
+        [JsonProperty("id")]
+        public string ProjectId { get; set; }
```

Comment:
This should not be apart of the JSON response, it should be apart of the method call.
```suggestion
        public string ProjectId { get; set; }
```